es-space-mgmt/main/default/lwc/spaceDesigner/spaceDesigner.js

```diff
@@ -1,48 +1,74 @@
+
+import TILE_SELECTION_MC from '@salesforce/messageChannel/Tile_Selection__c';
+import FLOW_STATUS_CHANGE_MC from '@salesforce/messageChannel/Flow_Status_Change__c';
+import {
+    subscribe,
+    unsubscribe,
+    APPLICATION_SCOPE,
+    MessageContext,
+    publish
+} from 'lightning/messageService';
 
 export default class SpaceDesigner extends LightningElement {
     /*
      *   Component coordinates event comms between Aura-based parent component
-     *   and LWC-based siblings. Uses pubsub in place of custom Aura application event.
+     *   and LWC-based siblings. Uses LMS in place of custom Aura application event.
      *   TO DO: Replace Aura parent component when support for LWC flow screens available.
      */
     flowStarted = false;
-    @wire(CurrentPageReference) pageRef;
+    subscription = null;
 
-    connectedCallback() {
-        registerListener(
-            'selectreservation',
-            this.handleReservationSelect,
-            this
+    @wire(MessageContext)
+    messageContext;
+
+    subscribeToMessageChannel() {
+        this.subscription = subscribe(
+            this.messageContext,
+            TILE_SELECTION_MC,
+            (message) => this.handleReservationSelect(message),
+            { scope: APPLICATION_SCOPE }
         );
     }
 
+    unsubscribeToMessageChannel() {
+        unsubscribe(this.subscription);
+        this.subscription = null;
+    }
+
+    connectedCallback() {
+        this.subscribeToMessageChannel();
+    }
+
     disconnectedCallback() {
-        unregisterAllListeners(this);
+        this.unsubscribeToMessageChannel();
     }
 
-    handleReservationSelect(event) {
-        if (!this.flowStarted) {
-            this.flowStarted = true;
-            this.dispatchEvent(
-                new CustomEvent('reservchoice', { detail: event.detail })
-            );
-        } else if (this.flowStarted) {
-            const toastEvt = new ShowToastEvent({
-                title: 'Flow interview already in progress',
-                message:
-                    'Finish the flow interview in progress before selecting another reservation.',
-                variant: 'error'
-            });
-            this.dispatchEvent(toastEvt);
+    handleReservationSelect(message) {
+        if (message.tileType === 'reservation') {
+            if (!this.flowStarted) {
+                this.flowStarted = true;
+                this.dispatchEvent(
+                    new CustomEvent('reservchoice', {
+                        detail: message.properties
+                    })
+                );
+            } else if (this.flowStarted) {
+                const toastEvt = new ShowToastEvent({
+                    title: 'Flow interview already in progress',
+                    message:
+                        'Finish the flow interview in progress before selecting another reservation.',
+                    variant: 'error'
+                });
+                this.dispatchEvent(toastEvt);
```

Comment:
very minor and I am probably nitpicking this. Looks like you have two different styles there. In one dispatchEvent you initialize an object and then use it in the subsequent statement and in another you out it directly. I think it will be awesome if we just go with one convention. I generally prefer more verbose like the `toastEvt` object initialization you have and then dispatching it.

Reply:
Will address in the next PR. There are 2 other files which have similar implementations